connector/dialect.tdd

```diff
@@ -384,6 +428,26 @@
       <argument type='real' />
       <argument type='int' />
     </function>
+    <function group='numeric' name='EXP' return-type='real'>
+      <formula>POW(2.718281828459046, %1)</formula>
+      <argument type='int' />
+    </function>
+    <function group='numeric' name='EXP' return-type='real'>
+      <formula>POW(2.718281828459046, %1)</formula>
+      <argument type='real' />
+    </function>
+    <function group='numeric' name='FLOOR' return-type='real'>
```

Comment:
I removed the function with the `int` arg and `real` return.  It makes sense that it would not be reachable.

As for returning int, from what I see in mongodb the type it returns matches the input type.  Maybe the documentation is not referring to the actual return type?
For example for $floor:
```
foo> db.ceilAndFloor.aggregate([ {$addFields : { "n": {$floor: "$v"}}}])
[
  { _id: 0, v: 7, n: 7 },
  { _id: 1, v: Long("7"), n: Long("7") },
  { _id: 2, v: -7.8, n: -8 }, <- this -8 is a Double type.
  { _id: 3, v: Decimal128("7.8"), n: Decimal128("7") }, <- Decimal output 
```

Reply:
oh, I see. Yes, the docs is not very clear on the type. That's fine then. Thanks for looking into this :)